internal/cmdplan/command.go

```diff
@@ -265,24 +264,82 @@ func printWithPrefix(text, prefix string, ioStreams genericclioptions.IOStreams)
 
 // printKRM outputs the plan inside a ResourceList so the output format
 // follows the KRM function wire format.
-func printKRM(plan *v1alpha1.Plan, ioStreams genericclioptions.IOStreams) error {
-	b, err := yaml.Marshal(plan)
+func printKRM(
+	plan *kptplanner.Plan,
+	ioStreams genericclioptions.IOStreams,
+) error {
+	planResource, err := yaml.Parse(strings.TrimSpace(`
+apiVersion: config.google.com/v1alpha1
```

Comment:
Should we use `kpt.dev` :)

Reply:
Agree. Updated to use `kpt.dev/v1alpha1`.